pandas/tests/indexes/timedeltas/test_astype.py

```diff
@@ -69,28 +53,7 @@ def test_astype_timedelta64(self):
         float, 'datetime64', 'datetime64[ns]'])
     def test_astype_raises(self, dtype):
         # GH 13149, GH 13209
-        idx = TimedeltaIndex([1e14, 'NaT', pd.NaT, np.NaN])
+        idx = TimedeltaIndex([1e14, 'NaT', NaT, np.NaN])
         msg = 'Cannot cast TimedeltaIndex to dtype'
         with tm.assert_raises_regex(TypeError, msg):
             idx.astype(dtype)
-
-    def test_pickle_compat_construction(self):
-        pass
-
-    def test_shift(self):
-        # test shift for TimedeltaIndex
-        # err8083
```

Comment:
note that this same test is also in `test_arithmetic.py`, and is overriden in `test_timedeltas`:

https://github.com/pandas-dev/pandas/blob/69cd5fbfd8b57033b0eb280ebc6dd502652f269c/pandas/tests/indexes/timedeltas/test_timedelta.py#L35-L36

https://github.com/pandas-dev/pandas/blob/69cd5fbfd8b57033b0eb280ebc6dd502652f269c/pandas/tests/indexes/timedeltas/test_arithmetic.py#L170-L187